Source/Extensions/Blazorise.Snackbar/wwwroot/blazorise.snackbar.css

```diff
@@ -1,7 +1,7 @@
 ﻿.snackbar {
   align-items: center;
   background-color: var(--b-snackbar-background, #323232);
-  color: var(--b-snackbar-text-color, white);
+  color: var(--b-snackbar-text-color, rgb(255, 255, 255));
```

Comment:
curious, any special reason for this apparent redudant change?

Reply:
@steal4life ?